git_repo/services/ext/gerrit.py

```diff
@@ -103,3 +103,20 @@ def request_create(self, onto_user, onto_repo, from_branch, onto_branch=None, ti
             'project': '/'.join([onto_user, onto_repo]),
             'remote': onto_branch
         }
+
+    def request_fetch(self, user, repo, request, pull=False, force=False): #pragma: no cover
```

Comment:
about `#pragma: no cover`, why are you excluding this function from coverage? As you've written tests for it, that can go :)

Reply:
Yes. It was placed by mistake. I will remove it now